Update bitmaps.md (#2561)
antirez/redis-doc: docs/data-types/bitmaps.md

* `SETBIT` sets a bit at the provided offset to 0 or 1.
* `GETBIT` returns the value of a bit at a given offset.

See the [complete list of bitmap commands](https://redis.io/commands/?group=bitmap).

## Example

Suppose you have 1000 cyclists racing through the country-side, labeled 0-999.

> You want to quickly determine whether a given rider has pinged a tracking server within the hour. 

Reviewer: Sorry, but I think connected sensors are more common than connected cyclists. Or maybe it's a censor on each bicycle? :)

You can represent this scenario using a bitmap whose key references the current hour.

> * Rider 123 pings the server on January 1, 2024 within the 00:00 hour. We can then confirm that rider 123 pinged the server. We can then check to see if rider 456 has pinged the server for that same hour.

Reviewer 2: ```suggestion
* Rider 123 pings the server on January 1, 2024 within the 00:00 hour. You can then confirm that rider 123 pinged the server. You can also check to see if rider 456 has pinged the server for that same hour.
```

{{< clients-example bitmap_tutorial ping >}}
> SETBIT pings:2024-01-01-00:00 123 1
(integer) 0
> GETBIT pings:2024-01-01-00:00 123
1
> GETBIT pings:2024-01-01-00:00 456
0

> {{< /clients-examples >}}

Reviewer 2: ```suggestion
{{< /clients-example >}}
```

3. `BITPOS` finds the first bit having the specified value of 0 or 1.

Both `BITPOS` and `BITCOUNT` are able to operate with byte ranges of the
string, instead of running for the whole length of the string. We can trivially see the number of bits that have been set in a bitmap.

{{< clients-example bitmap_tutorial bitcount >}}
> BITCOUNT pings:2024-01-01-00:00
(integer) 1

> {{< /clients-examples >}}

Reviewer 2: ```suggestion
{{< /clients-example >}}
```
